biogeochem/EDPatchDynamicsMod.F90

```diff
@@ -1225,51 +1257,6 @@ subroutine fire_litter_fluxes(currentSite, cp_target, new_patch_target, patch_si
           currentCohort => currentCohort%taller
 
        enddo  ! currentCohort
-
-       !************************************/     
-       ! PART 3) Burn parts of trees that did *not* die in the fire.
-       !         currently we only remove leaves. branch and assocaited sapwood consumption coming soon.
-       ! PART 4) Burn parts of grass that are consumed by the fire. 
-       ! grasses are not killed directly by fire. They die by losing all of their leaves and starving. 
-       !************************************/ 
-       currentCohort => new_patch%shortest
-       do while(associated(currentCohort))
-
-          sapw_c   = currentCohort%prt%GetState(sapw_organ, all_carbon_elements)
-          leaf_c   = currentCohort%prt%GetState(leaf_organ, all_carbon_elements)
-
-          call carea_allom(currentCohort%dbh,currentCohort%n,currentSite%spread,currentCohort%pft,currentCohort%c_area)
-
-          if(EDPftvarcon_inst%woody(currentCohort%pft) == 1)then
-             burned_leaves = leaf_c * currentCohort%fraction_crown_burned
-          else
-             burned_leaves = leaf_c * currentPatch%burnt_frac_litter(lg_sf)
-          endif
-
-          if (burned_leaves > 0.0_r8) then
-
-             ! Remove burned leaves from the pool
-             if(leaf_c>nearzero) then
-                leaf_burn_frac = burned_leaves/leaf_c
-             else
-                leaf_burn_frac = 0.0_r8
-             end if
-             call PRTBurnLosses(currentCohort%prt, leaf_organ, leaf_burn_frac)
-             
-             !KgC/gridcell/day
-             currentSite%flux_out = currentSite%flux_out + burned_leaves * currentCohort%n * &
-                  patch_site_areadis/currentPatch%area * AREA 
```

Comment:
Noobie question: I can't seem to find where `AREA` is defined in the code.  I noticed that you refactored the calculation to avoid using it and I wondered why.  Is it the variable defined by EDTypesMod?

Reply:
Looks like the definition in the module file is using different case:
https://github.com/NGEET/fates/blob/master/biogeochem/EDPatchDynamicsMod.F90#L10

Reply:
:man_facepalming: